server/settings/general.py

```diff
@@ -41,7 +41,7 @@ class GeneralSettingsModel(BaseSettingsModel):
         "houdini_vars": [
             {
                 "var": "JOB",
-                "value": "{root[work]}/{project[name]}/{hierarchy}/{asset}/work/{task[name]}",  # noqa
+                "value": "{root[work]}/{project[name]}/{hierarchy}/{folder[name]}/work/{task[name]}",  # noqa
```

Comment:
This is not backwards compatible, because it seems that the old `{asset}` resolved to {folder[path]}`? Or didn't it?

```suggestion
                "value": "{root[work]}/{project[name]}/{hierarchy}/{folder[path]}/work/{task[name]}",  # noqa
```

Also, note that `{folder[path]}` is prefixed with a `/` so technically the `/` in front of this doubles it. So it should be:

```suggestion
                "value": "{root[work]}/{project[name]}/{hierarchy}{folder[path]}/work/{task[name]}",  # noqa
```

Reply:
The folder path is equivalent for hierarchy/folder name. 
I've raised this question on Discord [here](https://discord.com/channels/517362899170230292/773127690453516298/1293236898680274995).

Reply:
You are absolutely correct - I missed `{hierarchy}` there.
Hmm - bit confused why there was a double forward slash in the logs then.

Reply:
Can't reproduce the slash issue myself.